src/@types/index.ts

```diff
@@ -0,0 +1,8 @@
+export type NameTheFrameSettings = {
```

Comment:
I wouldn't mention the name of the app everywhere.
```suggestion
export type Settings = {
```